Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -303,17 +302,20 @@ class CredentialsManagerSpec: QuickSpec {
                 }
             }
             
-            context("custom keychain", closure: {
+            context("custom keychain") {
+                let storage = A0SimpleKeychain(service: "test_service")
                 beforeEach {
-                    storage = A0SimpleKeychain(service: "test_service")
                     credentialsManager = CredentialsManager(authentication: authentication,
-                                                            storage: storage ?? A0SimpleKeychain())
+                                                            storage: storage)
                 }
                 
                 it("should accept custom keychains") {
-                    expect(storage?.service).to(match("test_service"))
+                    expect(credentialsManager.store(credentials: credentials)).to(beTrue())
```

Comment:
don't need to test the outcome here as we are testing for it through the data tests